Client/Frontend/Accessors/NewTabAccessors.swift

```diff
@@ -55,5 +55,22 @@ enum NewTabPage: String {
         }
     }
 
+    var url: NSURL? {
+        switch self {
+        case .BlankPage:
+            return nil
+        case .HomePage:
+            return nil
+        case .TopSites:
+            return NSURL(string:"#panel=0", relativeToURL: UIConstants.AboutHomePage)
```

Comment:
There are enums for these now, for ease of reference and ensuring that all home panels refer to the same thing. I created them for menu
enum:
 https://github.com/mozilla/firefox-ios/blob/ebde8f60379c3bd338c3e0153d7f7d6ef7df8073/Client/Frontend/Home/HomePanelViewController.swift#L50
usage example:
https://github.com/mozilla/firefox-ios/blob/f2386f93b7e7ed1026c24d935683f6b1470274dc/Client/Frontend/Browser/BrowserViewController.swift#L1330